### PR TITLE
[16.0][MIG] auth_admin_passkey: migration to 16.0

### PR DESCRIPTION
**Pull request by AnizR**
migration of auth_admin_passkey to version 16

### CODE REVIEW

File: auth_admin_passkey/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Authentification - System Administrator Passkey",
     "summary": "Allows system administrator to authenticate with any account",
-    "version": "14.0.1.0.1",
+    "version": "16.0.1.0.1",
```
Comment (rousseldenis) [inline auth_admin_passkey/__manifest__.py]: Could you reset version to `16.0.1.0.0`?

File: auth_admin_passkey/tests/test_auth_admin_passkey.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+from odoo import exceptions
+from odoo.tests import common, tagged
+from odoo.tools import config
+
+
+@tagged("post_install", "-at_install")
+class TestAuthAdminPasskey(common.TransactionCase):
+    """Tests for 'Auth Admin Passkey' Module"""
+
+    def setUp(self):
```
Comment (rousseldenis) [inline auth_admin_passkey/tests/test_auth_admin_passkey.py]: Could you convert this to class method ?

Comment (rousseldenis) [reply auth_admin_passkey/tests/test_auth_admin_passkey.py]: Could you also add this : 

`cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))`

File: auth_admin_passkey/tests/test_ui.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+from odoo import http
+from odoo.tests import common, tagged
+from odoo.tools import config
+
+
+@tagged("post_install", "-at_install")
+class TestUI(common.HttpCase):
+    def setUp(self):
+        super(TestUI, self).setUp()
```
Comment (rousseldenis) [inline auth_admin_passkey/tests/test_ui.py]: Same as above

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Comment by rousseldenis** [comment]
@adrienpeiffer Could you add your functional review ?

**Review by FrancoMaxime — APPROVED** [review]
Fonctionnaly tested.

**Review by alexis-via — APPROVED** [review]
Tested in real life

**Comment by AnizR** [comment]
Small security improvement: 
I was concerned with the fact that the password was written "in clear" in config file so, I added the possibility to store sha512 encrypted password in config file. 
(This is not activate by default but can be activated by setting 'auth_admin_passkey_password_sha512_encrypted' to true)

**Review by atchuthan — APPROVED** [review]
Tested functionally :+1:

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@OCA/core-maintainers

**Pull request merged**